web/styles/zulip.css

```diff
@@ -1332,8 +1332,8 @@ div.toggle_resolve_topic_spinner .loading_indicator_spinner {
 .empty-feed-notice-title {
     font-size: 1.5em;
     font-weight: 400;
-    line-height: 1;
     word-wrap: break-word;
+    line-height: 1.2;
```

Comment:
What impact does this have on existing banners? This might belong as its own commit with its own commit message, since it's not directly part of the feature (though may be required for it to look good), and impacts other components.